kf_update_dbgap_consent/sample_status.py

```diff
@@ -214,6 +220,48 @@ def entities_dict(endpoint, filt):
                 hidden_specimens[kfid] = bs
 
         """
+        Handle if samples loaded into dbgap should be made visible
+        """
+        if coerce_visible:
+            unhidden_specimens = {}
+            for kfid, bs in storage["biospecimens"].items():
+                sample = dbgap_samples.get(bs[match_entity], {})
+                if sample.get("@dbgap_status") == "Loaded":
+                    # remove the sample from list of hidden specimens if the
+                    # specimen is loaded in dbgap.
+                    hidden_specimens.pop(kfid, None)
+                    unhidden_specimens[kfid] = bs
+            descendants_of_unhidden_specimens = find_descendants_by_kfids(
+                self.db_url or self.api_url,
+                "biospecimens",
+                list(unhidden_specimens.keys()),
+                ignore_gfs_with_hidden_external_contribs=False,
```

Comment:
I think ignore_gfs_with_hidden_external_contribs should be True here. You don't want to unhide a multi-specimen GF that has other specimens in it that are still hidden. You want to follow the internal behavior of https://github.com/kids-first/kf-utils-python/blob/9ff5997566204a7375e8c68176d1e7e982f8b5c1/kf_utils/dataservice/descendants.py#L506